### PR TITLE
[IMP] web_calendar: assign default value to name field in calendar

### PR DESCRIPTION
**Pull request by rim-odoo**
- Task: https://www.odoo.com/web/#id=8785&view_type=form&model=project.task&menu_id=3942&action=327
- Pad: https://pad.odoo.com/p/r.CWkkEKeRTtbGnXpu

### CODE REVIEW

File: addons/web_calendar/static/src/js/web_calendar.js [hunk 1]
```diff
@@ -974,6 +976,12 @@ openerp.web_calendar = function(instance) {
             }
 
             self.$input = this.$el.find('input');
+            if ('name' in fields){
```
Comment (rga-odoo) [inline addons/web_calendar/static/src/js/web_calendar.js]: use underscore js

File: addons/web_calendar/static/src/js/web_calendar.js [hunk 2]
```diff
@@ -974,6 +976,12 @@ openerp.web_calendar = function(instance) {
...
+                var mod = new instance.web.Model(model, self.dataset.context, self.dataset.domain);
```
Comment (rga-odoo) [inline addons/web_calendar/static/src/js/web_calendar.js]: why create new Model ? we can use dateaset default_get method

File: addons/web_calendar/static/src/js/web_calendar.js (hunk 1 above)
Comment (avo-odoo) [inline addons/web_calendar/static/src/js/web_calendar.js]: Done

File: addons/web_calendar/static/src/js/web_calendar.js (hunk 2 above)
Comment (avo-odoo) [inline addons/web_calendar/static/src/js/web_calendar.js]: Done.

File: addons/web_calendar/static/src/js/web_calendar.js [hunk 3]
```diff
@@ -1007,7 +1007,12 @@ openerp.web_calendar = function(instance) {
                 self.trigger('close');
             })
             .open();
-            this.$input = this.$dialog.$('input').keyup(function enterHandler (e) {
+            self.$input = this.$dialog.$('input');
```
Comment (tde-banana-odoo) [inline addons/web_calendar/static/src/js/web_calendar.js]: Strange mixing the use of self.$input and this.$input. Please clean.

File: addons/web_calendar/static/src/js/web_calendar.js [hunk 4]
```diff
@@ -1007,7 +1007,12 @@ openerp.web_calendar = function(instance) {
...
+            if(_.has(this.getParent().fields, "name")){
```
Comment (tde-banana-odoo) [inline addons/web_calendar/static/src/js/web_calendar.js]: Please lint your code.

File: addons/web_calendar/static/src/js/web_calendar.js (hunk 3 above)
Comment (avo-odoo) [inline addons/web_calendar/static/src/js/web_calendar.js]: Done.

File: addons/web_calendar/static/src/js/web_calendar.js (hunk 4 above)
Comment (avo-odoo) [inline addons/web_calendar/static/src/js/web_calendar.js]: Done.

**Comment by rim-odoo** [comment]
@avo-odoo Can you fix the conflicts and rebase the branch? Thanks

**Pull request closed without merging**